app/src/main/java/com/gruppe11/todoApp/viewModel/ScheduleViewModel.kt

```diff
@@ -88,6 +88,9 @@ class ScheduleViewModel @Inject constructor(
 
     fun deleteTimeSlot(timeSlot: TimeSlot) {
         viewModelScope.launch {
+            timeSlot.tasks.forEach{
```

Comment:
I stedet for dette kunne du først slette timeslot og derefter hente alle tasks ned igen.